app/src/main/res/layout-sw600dp-land/revision_card_fragment.xml

```diff
@@ -0,0 +1,63 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.topic.revisioncard.RevisionCardViewModel" />
+  </data>
+
+  <ScrollView
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/oppiaLightGreen"
+    android:overScrollMode="never"
+    android:scrollbars="none">
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:gravity="center"
+      android:orientation="horizontal">
+
+      <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:minWidth="640dp"
+        android:orientation="vertical"
+        android:paddingBottom="145dp">
```

Comment:
The dimensions should be in multiples of `4dp` therefore I think we should make it 148dp

Reply:
Done